### PR TITLE
[Feature: Instructor UI] Instructor Edit Peer Grading Matrix

### PR DESCRIPTION
**Pull request by listeph**
### What is the current behavior?
Instructors need to manually edit the CSV and re-upload to edit peer grading assignments.
Screen Shot 2020-07-01 at 12 37 30 PM

### What is the new behavior?
Partially implements #5364 (minus the part that allows adding a new grader)
There is now a popup interface that allows adding/removing students to each grader, as well as removing an entire grader row. This is simpler for small edits instead of creating a new CVS. 
Screen Shot 2020-06-27 at 4 21 13 PM
Screen Shot 2020-06-27 at 4 21 38 PM
P.S. Only commits after May 20th are working on this PR (I branched off of something that wasn't master oops).

### CODE REVIEW

**Comment by bmcutler** [comment]
@listeph 
Please add a few screenshots to the top post of this PR.
These PR notes & images are incorporated in the release notes as documentation.
Thanks!

**Comment by emaicus** [comment]
Looking good so far. If you have any questions about AJAX and/or writing the query to update the peer grader set, let me know.

File: site/app/controllers/admin/AdminGradeableController.php [hunk 1]
```diff
@@ -264,8 +264,20 @@ private function editPage(Gradeable $gradeable, $semester, $course, $nav_tab = 0
...
         ]);
-        $this->core->getOutput()->renderOutput(['grading', 'ElectronicGrader'], 'popupStudents');
-        $this->core->getOutput()->renderOutput(['grading', 'ElectronicGrader'], 'popupMarkConflicts');
+        $this->core->getOutput()->renderOutput(array('grading', 'ElectronicGrader'), 'popupStudents');
+        $this->core->getOutput()->renderOutput(array('grading', 'ElectronicGrader'), 'popupMarkConflicts');
+        $this->core->getOutput()->renderOutput(array('admin', 'Gradeable'), 'AdminGradeablePeersForm', $gradeable);
+    }
+
+    /**
+     * @Route("/{_semester}/{_course}/gradeable/{gradeable_id}/update_peer_assignment", methods={"POST"})
```
Comment (MasterOdin) [inline site/app/controllers/admin/AdminGradeableController.php:273]:
```suggestion
     * @Route("/courses/{_semester}/{_course}/gradeable/{gradeable_id}/update_peer_assignment", methods={"POST"})
```

**Comment by bmcutler** [comment]
@listeph or @josharsh 

Please resolve the merge conflicts :)
Thanks

**Pull request merged**